python312Packages.cachecontrol: 0.14.0 -> 0.14.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python312Packages.cachecontrol is: Httplib2 caching for requests

meta.homepage for python312Packages.cachecontrol is: https://github.com/ionrock/cachecontrol

meta.changelog for python312Packages.cachecontrol is: https://github.com/psf/cachecontrol/releases/tag/v0.14.1


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 0.14.1 with grep in /nix/store/857n6kag88wbn1adzagd8jwivrrgjiqp-python3.12-cachecontrol-0.14.1
- found 0.14.1 in filename of file in /nix/store/857n6kag88wbn1adzagd8jwivrrgjiqp-python3.12-cachecontrol-0.14.1

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
74 total rebuild path(s)

74 package rebuild(s)

First fifty rebuilds by attrpath

adapta-gtk-theme
arc-theme
arx-libertatis
asciidoc-full
asciidoc-full-with-plugins
ayu-theme-gtk
breeze-hacked-cursor-theme
capitaine-cursors
catppuccin-cursors
cinnamon-screensaver
cinnamon.cinnamon-screensaver
cinnamon.iso-flags-png-320x420
clevis
coreboot-configurator
cwltool
dblatexFull
diagrams-as-code
disorderfs
emojione
fim
find-billy
hackneyed
hikounomizu
inkscape
inkscape-extensions.textext
inkscape-with-extensions
iso-flags
iso-flags-png-320x240
k40-whisperer
kakounePlugins.rep
luksmeta
mate.mate-panel-with-applets
mate.mate-utils
material-cursors
mojave-gtk-theme
numix-cursor-theme
numix-solarized-gtk-theme
pip-audit
plata-theme
poetry
poetryPlugins.poetry-audit-plugin
poetryPlugins.poetry-plugin-up
pop-gtk-theme
python311Packages.cachecontrol
python311Packages.cwl-upgrader
python311Packages.cwl-utils
python311Packages.diagrams
python311Packages.mkdocs-rss-plugin
python311Packages.orange-canvas-core
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/857n6kag88wbn1adzagd8jwivrrgjiqp-python3.12-cachecontrol-0.14.1 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python312Packages.cachecontrol https://github.com/r-ryantm/nixpkgs/archive/21ef47c65bfabb79d61a94734ce73344bd689bfb.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/21ef47c65bfabb79d61a94734ce73344bd689bfb#python312Packages.cachecontrol
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/857n6kag88wbn1adzagd8jwivrrgjiqp-python3.12-cachecontrol-0.14.1
ls -la /nix/store/857n6kag88wbn1adzagd8jwivrrgjiqp-python3.12-cachecontrol-0.14.1/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

nixpkgs-review took longer than 120m and timed out

---

###### Maintainer pings

cc @dotlambda for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

NickCao commented on pkgs/development/python-modules/cachecontrol/default.nix:
```diff
@@ -15,7 +15,7 @@
+  version = "0.14.1";
   pyproject = true;
 
   disabled = pythonOlder "3.7";
```
```suggestion
  disabled = pythonOlder "3.8";
```

dotlambda committed:
python312Packages.cachecontrol: 0.14.0 -> 0.14.1

Diff: https://github.com/ionrock/cachecontrol/compare/refs/tags/v0.14.0...v0.14.1

Changelog: https://github.com/psf/cachecontrol/releases/tag/v0.14.1

NickCao commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 354297`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>spring</li>
    <li>springLobby</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 167 packages built:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>arc-theme</li>
    <li>arx-libertatis</li>
    <li>asciidoc-full</li>
    <li>asciidoc-full-with-plugins</li>
    <li>asciidoc-full-with-plugins.dist</li>
    <li>asciidoc-full.dist</li>
    <li>ayu-theme-gtk</li>
    <li>breeze-hacked-cursor-theme</li>
    <li>capitaine-cursors</li>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>cinnamon-screensaver</li>
    <li>clevis</li>
    <li>clevis.man</li>
    <li>coreboot-configurator</li>
    <li>cwltool</li>
    <li>cwltool.dist</li>
    <li>dblatexFull</li>
    <li>diagrams-as-code</li>
    <li>diagrams-as-code.dist</li>
    <li>disorderfs</li>
    <li>emojione</li>
    <li>fim</li>
    <li>find-billy</li>
    <li>hackneyed</li>
    <li>hikounomizu</li>
    <li>inkscape</li>
    <li>inkscape-extensions.textext</li>
    <li>inkscape-extensions.textext.dist</li>
    <li>inkscape-with-extensions</li>
    <li>iso-flags</li>
    <li>iso-flags-png-320x240</li>
    <li>k40-whisperer</li>
    <li>kakounePlugins.rep</li>
    <li>luksmeta</li>
    <li>lyra-cursors</li>
    <li>mate.mate-panel-with-applets</li>
    <li>mate.mate-utils</li>
    <li>material-cursors</li>
    <li>mojave-gtk-theme</li>
    <li>numix-cursor-theme</li>
    <li>numix-solarized-gtk-theme</li>
    <li>pip-audit</li>
    <li>pip-audit.dist</li>
    <li>plata-theme</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>pop-gtk-theme</li>
    <li>python311Packages.cachecontrol</li>
    <li>python311Packages.cachecontrol.dist</li>
    <li>python311Packages.cwl-upgrader</li>
    <li>python311Packages.cwl-upgrader.dist</li>
    <li>python311Packages.cwl-utils</li>
    <li>python311Packages.cwl-utils.dist</li>
    <li>python311Packages.diagrams</li>
    <li>python311Packages.diagrams.dist</li>
    <li>python311Packages.mkdocs-rss-plugin</li>
    <li>python311Packages.mkdocs-rss-plugin.dist</li>
    <li>python311Packages.orange-canvas-core</li>
    <li>python311Packages.orange-canvas-core.dist</li>
    <li>python311Packages.orange-widget-base</li>
    <li>python311Packages.orange-widget-base.dist</li>
    <li>python311Packages.orange3</li>
    <li>python311Packages.orange3.dist</li>
    <li>python311Packages.osc-diagram</li>
    <li>python311Packages.osc-diagram.dist</li>
    <li>python311Packages.pywebcopy</li>
    <li>python311Packages.pywebcopy.dist</li>
    <li>python311Packages.schema-salad</li>
    <li>python311Packages.schema-salad.dist</li>
    <li>python312Packages.cachecontrol</li>
    <li>python312Packages.cachecontrol.dist</li>
    <li>python312Packages.cwl-upgrader</li>
    <li>python312Packages.cwl-upgrader.dist</li>
    <li>python312Packages.cwl-utils</li>
    <li>python312Packages.cwl-utils.dist</li>
    <li>python312Packages.diagrams</li>
    <li>python312Packages.diagrams.dist</li>
    <li>python312Packages.mkdocs-rss-plugin</li>
    <li>python312Packages.mkdocs-rss-plugin.dist</li>
    <li>python312Packages.orange-canvas-core</li>
    <li>python312Packages.orange-canvas-core.dist</li>
    <li>python312Packages.orange-widget-base</li>
    <li>python312Packages.orange-widget-base.dist</li>
    <li>python312Packages.orange3</li>
    <li>python312Packages.orange3.dist</li>
    <li>python312Packages.osc-diagram</li>
    <li>python312Packages.osc-diagram.dist</li>
    <li>python312Packages.pywebcopy</li>
    <li>python312Packages.pywebcopy.dist</li>
    <li>python312Packages.schema-salad</li>
    <li>python312Packages.schema-salad.dist</li>
    <li>sherlock</li>
    <li>sherlock.dist</li>
    <li>tang</li>
    <li>tang.man</li>
    <li>vimix-cursors</li>
    <li>volantes-cursors</li>
    <li>wpa_supplicant_gui</li>
  </ul>
</details>
